packages/atlas-service/src/main.spec.ts

```diff
@@ -147,7 +149,8 @@ describe('AtlasServiceMain', function () {
       try {
         await AtlasService.getQueryFromUserPrompt({
           userPrompt: 'test',
-          collectionName: 'test.test',
+          collectionName: 'test',
+          databaseName: 'peanut',
           sampleDocuments: [{ test: '4'.repeat(60000) }],
```

Comment:
Should we remove sampleDocuments as we were recently discussing?

Reply:
The caller of this function doesn't pass the `sampleDocuments` currently, and we have a regression test to make sure they aren't passed: https://github.com/mongodb-js/compass/pull/4671/files#diff-4cd0f8f72aa57907ac7a7876d0f2c423e04fe5e78d19e043e88aeba548ec1ebdR72
So I think it's fine to leave this part here as we might add it back in for some testing later on. For the same reason we left the sample documents code in the backend.
If you feel strongly about removing them from this function's interface I'm happy to.

Reply:
Since we have tests for this, I think it is fine to keep it here.